livewordcloud.py

```diff
@@ -80,7 +85,7 @@ def multi_color_func(word=None,
 
     img = io.BytesIO()
 
-    # savefig will save the figure (at resolution 300dpi - good enoough for print)
+    # savefig will save the figure (resolution 300dpi - good enoough for print)
```

Comment:
Funny that you removed "at" but kept "enoough" :)

Reply:
haha I was looking to drop 2 characters and I saw 'at' :D